WordPress/Classes/Utility/ContextManager.swift

```diff
@@ -8,8 +8,25 @@ let ContextManagerModelNameCurrent = "$CURRENT"
 
 public protocol CoreDataStackSwift: CoreDataStack {
 
+    /// Execute the given block with a background context and save the changes _if the block does not throw an error_.
+    ///
+    /// This function _does not block_ its running thread. The block is executed in background and the its return value
+    /// (or an error) is passed onto the `completion` block which is executed on the given `queue`.
+    ///
+    /// - Parameters:
+    ///   - block: A closure that uses the given `NSManagedObjectContext` to make Core Data model changes. The changes
+    ///         are only saved if the block does not throw an error.
+    ///   - completion: A closure which is called with the `block`'s execution result, which is either an error thrown
+    ///         by the `block` or the return value of the `block`.
+    ///   - queue: A queue on which to execute the completion block.
     func performAndSave<T>(_ block: @escaping (NSManagedObjectContext) throws -> T, completion: ((Result<T, Error>) -> Void)?, on queue: DispatchQueue)
 
+    /// Execute the given block with a background context and save the changes _if the block does not throw an error_.
+    ///
+    /// - Parameter block: A closure that uses the given `NSManagedObjectContext` to make Core Data model changes.
+    /// The changes are only saved if the block does not throw an error.
```

Comment:
Nitpick: How do you feed about indenting the second line?

```suggestion
    /// - Parameter block: A closure that uses the given `NSManagedObjectContext` to make Core Data model changes.
    ///   The changes are only saved if the block does not throw an error.
```

No strong feelings. Xcode renders both versions as one would expect:

<img width="1442" alt="Screenshot 2023-03-08 at 2 34 32 pm" src="https://user-images.githubusercontent.com/1218433/223613180-ba4fda06-2128-4e88-96c4-90343e17e474.png">

Reply:
Thanks! All of above are addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/20279/commits/a0b6b61fbc1f6af10b78aaa5d10c4d00da9d00dd